.github/workflows/conda.yml

```diff
@@ -0,0 +1,32 @@
+name: Setup Anaconda environment
+
+on:
+  pull_request:
+    branches:
```

Comment:
One more thing, could you config this workflow only run when the `environment.yml` file changes?

Reply:
I briefly looked into it earlier and I don't think you can, I'll look into it though